### PR TITLE
SWAT suit no longer has space-proofing, less slowdown and retains temp resist, cargo cost lowered to 1400

### PR DESCRIPTION
**Pull request by castawaynont**
## About The Pull Request
The SWAT suit ordered from cargo is no longer space-proof, but it's slowdown is cut back. It retains it's temperature proofing. It's crate cost from cargo has been reduced to 1400 from 2400.

## Why It's Good For The Game
From the precedent of MODsuits, choosing between space-proofness and having armour should be an active decision. This PR brings it in line with other crew suits. This is a pretty hefty nerf, so I felt it reasonable to cut down it's slowdown down to 0.7, from 1 (same as an EVA suit), as well as keep it's temperature resist. It's price has also been dropped down for the same reason.

## Changelog

:cl: 
balance: CentComm's Mark One SWAT suit's design has been reworked; it now lacks spaceproofing, but it has become significantly faster to move in and retains temperature-proofing. These adjustments have allowed CC to sell them for a lower cost of 1400.
/:cl:

### CODE REVIEW

**Comment by Serxule** [comment]
i always thought the space-proof was useless

**Comment by SuperNovaa41** [comment]
you messed up your changelog put it on the line AFTER the 🆑

File: code/modules/clothing/spacesuits/swat.dm [hunk 1]
```diff
@@ -1,9 +1,16 @@
 /obj/item/clothing/suit/space/swat
```
Comment (JohnFulpWillard) [inline code/modules/clothing/spacesuits/swat.dm:1]: Why not move it off of a space suit subtype

Comment (castawaynont) [reply code/modules/clothing/spacesuits/swat.dm:1]: so just stick it into armor.dm?

Comment (castawaynont) [reply code/modules/clothing/spacesuits/swat.dm]: moved into armor.dm

File: code/modules/clothing/suits/armor.dm [hunk 2]
```diff
@@ -228,6 +228,22 @@
...
+	name = "MK.I SWAT Suit"
+	desc = "A tactical suit first developed in a joint effort by the defunct IS-ERI and Nanotrasen in 2321 for military operations. It has a minor slowdown, but offers decent protection."
+	icon_state = "heavy"
+	inhand_icon_state = "swat_suit"
+	armor = list(MELEE = 40, BULLET = 30, LASER = 30,ENERGY = 40, BOMB = 50, BIO = 90, FIRE = 100, ACID = 100, WOUND = 15)
+	strip_delay = 120
+	resistance_flags = FIRE_PROOF | ACID_PROOF
+	clothing_flags = THICKMATERIAL
+	min_cold_protection_temperature = SPACE_SUIT_MIN_TEMP_PROTECT
+	max_heat_protection_temperature = SPACE_SUIT_MAX_TEMP_PROTECT
```
Comment (Fikou) [inline code/modules/clothing/suits/armor.dm:243]: i think you have to set heat and cold protection's bodyparts

File: code/modules/clothing/head/helmet.dm [hunk 3]
```diff
@@ -239,9 +239,16 @@
...
 	name = "\improper SWAT helmet"
-	desc = "An extremely robust, space-worthy helmet with the Nanotrasen logo emblazoned on the top."
+	desc = "An extremely robust helmet with the Nanotrasen logo emblazoned on the top."
 	icon_state = "swat"
 	inhand_icon_state = "swat"
+	clothing_flags = PLASMAMAN_HELMET_EXEMPT
+	cold_protection = HEAD
+	min_cold_protection_temperature = SPACE_HELM_MIN_TEMP_PROTECT
+	heat_protection = HEAD
+	max_heat_protection_temperature = SPACE_HELM_MAX_TEMP_PROTECT
```
Comment (Fikou) [inline code/modules/clothing/head/helmet.dm:249]: are these not inherited from parent

Comment (castawaynont) [reply code/modules/clothing/head/helmet.dm:249]: It doesn't seem to work without these so probably

**Comment by ATH1909** [comment]
without the spaceproofing, this is definitely not worth the slowdown if red slime products aren't in play

**Comment by castawaynont** [comment]
> without the spaceproofing, this is definitely not worth the slowdown if red slime products aren't in play

the slowdown is like 0.7 now, pretty fast still

**Comment by MacBlaze1** [comment]
> > without the spaceproofing, this is definitely not worth the slowdown if red slime products aren't in play
> 
> the slowdown is like 0.7 now, pretty fast still

Unfortunately, that's still effectively an emergency firesuit worth of slowdown.

**Comment by castawaynont** [comment]
> > > without the spaceproofing, this is definitely not worth the slowdown if red slime products aren't in play
> > 
> > 
> > the slowdown is like 0.7 now, pretty fast still
> 
> Unfortunately, that's still effectively an emergency firesuit worth of slowdown. Writing significantly faster in your PR description is either disingenuous or unaware of how slowdown is scaled between items on TG.
 It IS significantly faster from the previous version's slowdown of 1. You're welcome to try out the speed difference yourself

**Comment by ErdinyoBarboza** [comment]
Without its inherent spaceproofness this armor becomes useless as the slowdown is not worthwhile the added armor it has over a standard issue armor.

**Comment by Valorium-RB** [comment]
Behold, yet another nail in the coffin of Security being able to do anything against nuclear operatives

**Comment by optimumtact** [comment]
> Behold, yet another nail in the coffin of Security being able to do anything against nuclear operatives

Nuclear operatives has a win rate of less than 30%

**Comment by castawaynont** [comment]
@optimumtact https://moth.fans/data/nukes-n-flukes/

**Comment by necromanceranne** [comment]
Ops kinda need help I suppose.

**Comment by Bawhoppen** [comment]
I think my stance on this has been quite clear

🥇 🥇 🥇 

(999)

**Comment by Bawhoppen** [comment]
@tralezab you have betrayed me, how could you do this Armalezab

**Comment by optimumtact** [comment]
RIP swat

**Comment by ATH1909** [comment]
> > > > without the spaceproofing, this is definitely not worth the slowdown if red slime products aren't in play
> > > 
> > > 
> > > the slowdown is like 0.7 now, pretty fast still
> > 
> > 
> > Unfortunately, that's still effectively an emergency firesuit worth of slowdown. Writing significantly faster in your PR description is either disingenuous or unaware of how slowdown is scaled between items on TG.
> 
> It IS significantly faster from the previous version's slowdown of 1. You're welcome to try out the speed difference yourself

okay but the slowdown is still shit compared to normal armor

**Comment by TJatPBnJ** [comment]
honk shooooo

**Pull request merged**